### PR TITLE
Add financial contributors

### PR DESCRIPTION
**Pull request by alex-w**
### Description
This pull request adding a new section into Help/About window - "Financial contributors" - with small introduction and short list of best financial contributors.

### Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] This change requires a documentation update

## Checklist:

- [x] My code follows the [code style](http://stellarium.org/doc/head/codingStyle.html) of this project.
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation (header file)
- [ ] I have updated the respective chapter in the Stellarium User Guide
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: src/gui/HelpDialog.cpp [hunk 1]
```diff
@@ -504,6 +504,19 @@ void HelpDialog::updateAboutText(void) const
 		     "Atque", "Worachate Boonplod"});
 	contributors.sort();
 
+	typedef QPair<QString, int> donator;
+	QVector<donator> financialContributors = {
+		{ "Laurence Holt", 1000 }, { "John Bellora", 470 }, { "Vernon Hermsen", 324 },
```
Comment (gzotti) [inline src/gui/HelpDialog.cpp:509]: I am not sure the actual numbers should be listed. People may not even want that (we did not ask them before!). We can list the people e.g. alphabetically or sorted by amount.

File: src/gui/HelpDialog.cpp [hunk 2]
```diff
@@ -558,6 +571,10 @@ void HelpDialog::updateAboutText(void) const
 	newHtml += "<li>" + q_("Tester: %1").arg(QString("Khalid AlAjaji")).toHtmlEscaped() + "</li></ul>";
 	newHtml += "<h3>" + q_("Contributors").toHtmlEscaped() + "</h3>";
 	newHtml += "<p>"  + q_("Several people have made contributions to the project and their work has made Stellarium better (sorted alphabetically): %1.").arg(contributors.join(", ")).toHtmlEscaped() + "</p>";
+	newHtml += "<h3>" + q_("Financial contributors").toHtmlEscaped() + "</h3>";
+	newHtml += "<p>"  + q_("Many people and organizations are supporting the development the Stellarium by the money and the best financial contributors (with donations $250+ USD) are %1.").arg(bestFinancialContributors.join(", ")).toHtmlEscaped();
```
Comment (gzotti) [inline src/gui/HelpDialog.cpp:575]: Many individuals and organizations are supporting the development of Stellarium by donations, and the most generous financial contributors (with donations of $250 or more) are %1.

File: src/gui/HelpDialog.cpp [hunk 3]
```diff
@@ -504,6 +504,19 @@ void HelpDialog::updateAboutText(void) const
...
+	QVector<donator> financialContributors = {
+		{ "Laurence Holt", 1000 }, { "John Bellora", 470 }, { "Vernon Hermsen", 324 },
+		{ "Satish Mallesh", 260 }, { "Marla Pinaire", 260 }, { "Vlad Magdalin", 250  },
+		{ "Astronomie-Werkstatt \"Sterne ohne Grenzen\"", 520 }, { "Triplebyte", 280 }
+	};
+	qSort(financialContributors.begin(), financialContributors.end(), [](donator i, donator j){ return i.second > j.second; });
+	QStringList bestFinancialContributors;
+	for (auto fc = financialContributors.begin(); fc != financialContributors.end(); ++fc)
+	{
+		bestFinancialContributors << QString("%1 ($%2 USD)").arg(fc->first, QString::number(fc->second));
```
Comment (gzotti) [inline src/gui/HelpDialog.cpp:517]: OK, did not see the sort before :-) But I would omit the actual numbers here.

**Comment by alex-w** [comment]
Done.

**Review by gzotti — APPROVED** [review]
OK for me!

**Pull request merged**